test/components/TopNav/NavMenuTest.js

```diff
@@ -1,7 +1,8 @@
 'use strict'
 import { expect } from 'chai'
 import React from 'react'
-import createComponent from 'test/helpers/shallowRenderHelper'
+import { shallow } from 'enzyme'
```

Comment:
I investigated using Enzyme to inject state into the component using its `setState` method which only works for class-based components. I like the API which it provides, though I think these tests _could_ be implemented without it, I believe it will be useful in TDD-ing components.